### PR TITLE
add NumFOCUS sponsorship (issue #2460)

### LINKED ISSUE #2460: indicate NF sponsorship
Related to MDAnalysis/MDAnalysis.github.io#115

- [x] update README: “Powered by NumFOCUS” badge for your website and/or GitHub - Markdown for adding NumFOCUS badge to project websites:  [Powered by NumFOCUS](http://numfocus.org)
   ```
   [![Powered by NumFOCUS](https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A )](http://numfocus.org)
   ```
- [x] add a Fiscal Sponsor Readme Attribution to the README, following https://github.com/numfocus/templates/blob/master/fiscal-sponsor-readme-attribution.md
 - [x] add text to the docs' top page

### PR DESCRIPTION
**Pull request by richardjgowers**
Fixes #2460

Changes made in this Pull Request:
 - 

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

**Comment by richardjgowers** [comment]
So on the [template for numfocus attribution](https://github.com/numfocus/templates/blob/master/fiscal-sponsor-readme-attribution.md) there's a [little thing](https://github.com/numfocus/templates/blob/master/fiscal-sponsor-readme-attribution.md#more) that they use for automatic audits.  This is in markdown, whereas our readme is rst, so it look like a quick rejig into markdown is required... unless anyone objects?

**Comment by IAlibay** [comment]
Not that it adds much to the discussion, it is interesting that a thrawl through github doesn't seem to indicate any cases of this auditing tag being used anywhere (I was having a look through to see if anyone came up with an alternative style of `.rst` files).

Considering this template is new as of last September, I assume this isn't something they have fully implemented yet?

**Comment by jbarnoud** [comment]
> 
> 
> So on the [template for numfocus attribution](https://github.com/numfocus/templates/blob/master/fiscal-sponsor-readme-attribution.md) there's a [little thing](https://github.com/numfocus/templates/blob/master/fiscal-sponsor-readme-attribution.md#more) that they use for automatic audits. This is in markdown, whereas our readme is rst, so it look like a quick rejig into markdown is required... unless anyone objects?

Could rst comments be of any help? https://docutils.sourceforge.io/docs/user/rst/quickref.html#comments

If NumFOCUS just search for the string, then it may be enough to have it appear in the readme somehow.

File: README.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@
   MDAnalysis Repository README
 ================================
 
-|build| |cov| [*]_
+|numfocus| |build| |cov| [*]_
```
Comment (orbeckstASU) [inline README.rst]: looks good

Comment (orbeckstASU) [reply README.rst]: So even if I select a "changes requested", it will automatically turn my review in the "comment" type. That's fair.

Comment (orbeckstASU) [reply README.rst]: @richardjgowers please ignore this whole review. It's a test but I haven't figure out how to remove it.

**Comment by orbeckst** [comment]
> So on the template for numfocus attribution there's a little thing that they use for automatic audits. This is in markdown, whereas our readme is rst, so it look like a quick rejig into markdown is required... unless anyone objects?

Raise an issue at https://github.com/numfocus/templates/issues and ask for how to use in rst ?

File: README.rst [hunk 2]
```diff
@@ -70,11 +70,26 @@ guide_. If you are a new developer who would like to start contributing to
...
 to find uncovered code.
 
+
+
+The MDanalysis project uses an `open governance model` and is fiscally sponsored by `NumFOCUS`_. Consider making 
+a `tax-deductible donation`_ to help the project pay for developer time, professional services, travel, workshops, and a variety of other needs.
+
+.. image:: https://raw.githubusercontent.com/numfocus/templates/master/images/numfocus-logo.png
+  :scale: 1 %
+  :target: <PROJECT_NUMFOCUS_SPONSORSHIP_LINK>
```
Comment (orbeckst) [inline README.rst]: Do we want to wait with merging until we have the link?

Comment (richardjgowers) [reply README.rst]: probably yep

**Comment by orbeckst** [comment]
I'll add the link from #2506

File: README.rst (hunk 2 above)
Comment (orbeckst) [reply README.rst]: added links etc

**Comment by orbeckst** [comment]
I added the links and did some general editing. However, two things are still bothering me
- I am not sure that the resizing of the numfocus image actually works, at least not in the preview. According to the template it should be a height of 60px.
- I don't know what to do with the magic markdown audit code `[//]: # (numfocus-fiscal-sponsor-attribution)` from the template.

**Comment by jbarnoud** [comment]
> I don't know what to do with the magic markdown audit code [//]: # (numfocus-fiscal-sponsor-attribution) from the template.

Maybe we should just ask numfocus?

**Comment by orbeckst** [comment]
Yes, will do.

**Comment by orbeckst** [comment]
Hm, apparently we ought to have a GOVERNANCE.md in the repo, even if it's only a stub, so will add one...

- If your project has no GOVERNANCE.md at the root, then create a stub GOVERNANCE.md and include a link to your governance documentation or the appropriate parties to contact for more information.

**Pull request merged**